README.md

```diff
@@ -1,25 +1,53 @@
-cgpNgsQc
-========
+# cgpNgsQc
 
 Collection of code for checking NSG sequencing results.
 
-[![Build Status](https://travis-ci.org/cancerit/cgpNgsQc.svg?branch=master)](https://travis-ci.org/cancerit/cgpNgsQc) : master  
-[![Build Status](https://travis-ci.org/cancerit/cgpNgsQc.svg?branch=develop)](https://travis-ci.org/cancerit/cgpNgsQc) : develop
+| Master                                        | Develop                                         |
+| --------------------------------------------- | ----------------------------------------------- |
+| [![Master Badge][travis-master]][travis-base] | [![Develop Badge][travis-develop]][travis-base] |
 
-Dependancies
-============
+Contents:
+
+<!-- TOC depthFrom:2 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Usage](#usage)
+- [Dependancies](#dependancies)
+- [Docker, Singularity and Dockstore](#docker-singularity-and-dockstore)
+- [LICENCE](#licence)
+
+<!-- /TOC -->
+
+## Usage
+
+Please see the [wiki](https://travis-ci.org/cancerit/cgpNgsQc/wiki) for details of usage.
```

Comment:
This isn't the right link. Possibly meant to be [here?](https://github.com/cancerit/cgpNgsQc/wiki) `https://github.com/cancerit/cgpNgsQc/wiki`